bundle/Templating/Twig/Extension/SiteRuntime.php

```diff
@@ -36,18 +41,24 @@ class SiteRuntime
 
     protected ?RemoteMediaProvider $remoteMediaProvider;
 
+    protected LoggerInterface $logger;
+
     public function __construct(
+        string $environment,
         PathHelper $pathHelper,
         LocaleConverterInterface $localeConverter,
         LoadService $loadService,
         VariationHandler $imageVariationService,
-        ?RemoteMediaProvider $remoteMediaProvider = null
+        ?RemoteMediaProvider $remoteMediaProvider = null,
+        ?LoggerInterface $logger = null
     ) {
+        $this->environment = $environment;
```

Comment:
I'd rather we inject `%kernel.debug%` parameter, which is a boolean, which we can call here `$isDebug` e.g.

Reply:
Changed in 09fa7f0